src/main/resources/edu/cornell/kfs/krad/config/CUKRADSpringBeans.xml

```diff
@@ -63,12 +63,4 @@
     <bean id="maintenanceDocumentDao" parent="platformAwareDao"
           class="edu.cornell.kfs.krad.dao.impl.CuMaintenanceDocumentDaoOjb"/>
 
-    <bean class="edu.cornell.kfs.krad.service.impl.CuMaintenanceDocumentServiceImpl"
```

Comment:
If this service class gets added back in, then this bean override will need to be added back in as well.